indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -149,14 +149,18 @@
             var $breadcrumbs = $('<ul>', {class: 'breadcrumbs'});
             var tag = clickable ? '<a>' : '<span>';
 
-            _.each(path, function(category) {
-                var $item = $(tag, {text: category.title});
+            _.each(path, function(category, idx) {
+                var $item = $('<li>');
```

Comment:
Maybe a `<li>` element is not the best option here, at least semantically. But not a strong opinion, since we also add the separator inside the `<ul>`.